client/dom/ssmLink.ts

```diff
@@ -0,0 +1,33 @@
+/*
+Make html links from ssm (simple somatic mutation) urls
+
+Arguments:
+    - ssm_urls: ssm url(s) specified in dataset file, can be either array or object
+    - m: m element from snp.mlst
+
+Returns: array of html links (if ssm_urls is array) or html string (if ssm_urls is object)
+*/
+export function makeSsmLink(ssm_urls: object | object[], m: object, genome: string) {
+	if (Array.isArray(ssm_urls)) {
+		// ssm_urls is array
+		const htmls = []
+		if (m.vcf_id) {
```

Comment:
change logic:
for(const ssmUrl of ssm_urls) {
  if(ssmUrl.namekey=='chr:pos') {
     // hardcoded logic to compose regulomedb
  } else {
    // value of m[ssmUrl.namekey] is expected to exist. if so append it to ssmUrl.base to compose. otherwise won't make url
  }
}

Reply:
the code will never detect any literal attribute as m.vcf_id